webapp/src/components/ActivityPage/ActivityPage.css

```diff
@@ -26,6 +26,10 @@
   margin-bottom: 30px;
 }
 
+.ActivityPage .clear-button {
+  text-align: right !important;
```

Comment:
This button is not aligned to the title that should the number of Activities. It can be fixed with:
```
.ActivityPage .clear-button {
    text-align: right !important;
    float: right;
}
```

Reply:
I wanted to avoid using that because aligment of full component is going left like this:

before:
![image](https://user-images.githubusercontent.com/14364153/41805092-bfb8ab30-7678-11e8-8083-a895f3d3266f.png)

after:
![image](https://user-images.githubusercontent.com/14364153/41805096-cf97e444-7678-11e8-92d0-04a98dbce016.png)

Still think that's ok?